assets/js/src/core/components/preview-card/preview-card.tsx

```diff
@@ -50,46 +52,54 @@ export const PreviewCard = (props: PreviewCardProps): React.JSX.Element => {
   }
 
   return (
-    <Card
-      className={ [styles.card, classCard].join(' ') }
-      cover={
-        <div className={ classImgDiv }>
-          <IconOrImage
-            alt={ props.name }
-            class={ classImg }
-            value={ props.imgSrc! }
-          />
-        </div>
-        }
-      onClick={ (event) => {
-        if (
-          dropdownMenuRef.current === null ||
-          dropdownMenuRef.current.menu?.list.contains(event.target as Node) === false
-        ) {
-          props.onClick?.(event)
-        }
-      } }
+    <Tooltip
+      open={ isHovering }
+      placement={ 'right' }
+      title={ props.name }
     >
-      <Dropdown
-        menu={ {
-          items: props.dropdownItems
+      <Card
+        className={ [styles.card, classCard].join(' ') }
+        cover={
+          <div className={ classImgDiv }>
+            <IconOrImage
+              alt={ props.name }
+              class={ classImg }
+              value={ props.imgSrc! }
+            />
+          </div>
+          }
+        onClick={ (event) => {
+          if (
+            dropdownMenuRef.current === null ||
+            dropdownMenuRef.current.menu?.list.contains(event.target as Node) === false
+          ) {
+            props.onClick?.(event)
+          }
         } }
-        menuRef={ dropdownMenuRef }
-        placement='bottomLeft'
+        onMouseEnter={ () => { setIsHovering(true) } }
```

Comment:
Why do we need to add our own hover state handling, shouldn't this be done automatically by the tooltip component when hovering over a contained item?

Reply:
absolutely correct - how dumb :P dont even know why i did that :D